openfisca_core/entities.py

```diff
@@ -281,15 +285,15 @@ def init_from_json(self, entities_json):
 
         Entity.init_from_json(self, entities_json)
```

Comment:
At this stage, the simulation is not yet fully initialised. We are in the process of building a "group" entity, and this is part of building the simulation.

We can (and need to) call `self.simulation.persons` because by construction, the "persons" entity is [always the first entity to be built](https://github.com/openfisca/openfisca-core/blob/7414070d634bef4040ec6e4ed246695f77bbfc16/openfisca_core/simulations.py#L88). We actually need to know data about the person entity to be able to build the group entities.